### PR TITLE
[MM-60809] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/admin_console/license_settings/trial_license_card/trial_license_card.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by Gesare5**
#### Summary

Replaces FormattedMarkdownMessage with FormattedMessage in trial_license_card.tsx
Updates en.json

#### Ticket Link

Fixes  [#28471](https://github.com/mattermost/mattermost/issues/28471)
Jira  [https://mattermost.atlassian.net/browse/MM-60809](https://mattermost.atlassian.net/browse/MM-60809)

#### Screenshots

Screenshot2

Screenshot3

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/license_settings/trial_license_card/trial_license_card.tsx [hunk 1]
```diff
@@ -28,21 +27,23 @@ const TrialLicenseCard: React.FC<Props> = ({license}: Props) => {
...
                 />
             );
         }
 
         return (
-            <FormattedMarkdownMessage
-                id='admin.license.trialCard.description'
-                defaultMessage='Your free trial will expire in **{daysCount} {daysCount, plural, one {day} other {days}}**. Visit our customer portal to purchase a license now to continue using Mattermost Professional and Enterprise features after trial ends.'
+            <FormattedMessage
+                id='trialLicenseCard.description'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/license_settings/trial_license_card/trial_license_card.tsx:43]: nit `trialLicenseCard.expiringAfterFewDays`

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/admin_console/license_settings/trial_license_card/trial_license_card.tsx [hunk 2]
```diff
@@ -28,21 +27,23 @@ const TrialLicenseCard: React.FC<Props> = ({license}: Props) => {
...
         if (currentDate.toDateString() === endDate.toDateString()) {
             return (
-                <FormattedMarkdownMessage
-                    id='admin.license.trialCard.description.expiringToday'
-                    defaultMessage='Your free trial expires **Today at {time}**. Visit our customer portal to purchase a license now to continue using Mattermost Professional and Enterprise features after trial ends'
+                <FormattedMessage
+                    id='admin.license.trialLicenseCard.expiringToday'
+                    defaultMessage='Your free trial expires <b>Today at {time}</b>. Visit our customer portal to purchase a license now to continue using Mattermost Professional and Enterprise features after trial ends'
                     values={{
+                        b: (chunks: string) => <b>{chunks}</b>,
```
Comment (larkox) [inline webapp/channels/src/components/admin_console/license_settings/trial_license_card/trial_license_card.tsx:34]: @M-ZubairAhmed Same question about `<b>` vs `<strong>` as in the other PR.

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**